util/admin/admin.go

```diff
@@ -134,6 +134,21 @@ func GetDDLJobs(txn kv.Transaction) ([]*model.Job, error) {
 	return jobs, nil
 }
 
+// GetDDLJobs returns the DDL jobs and an error.
+func GetDDLJobs(txn kv.Transaction) ([]*model.Job, error) {
+	t := meta.NewMeta(txn)
+	generalJobs, err := getDDLJobsInQueue(t)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	t.SetJobListKey(meta.AddIndexJobListKey)
+	addIdxJobs, err := getDDLJobsInQueue(t)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	return append(generalJobs, addIdxJobs...), nil
```

Comment:
The return jobs may be not sorted by job ID, should we return sorted jobs? Because the older function naturally return a sorted jobs

Reply:
I think it can be done in the next PR. I added a "TODO" now.